### ISSUE TITLE
`comma-spacing` has conflict with `space-in-parens`

### ISSUE THREAD

**Issue by g-plane** [eslint/eslint#11295]
**Tell us about your environment**

* **ESLint Version:** 5.12.1

**What parser (default, Babel-ESLint, etc.) are you using?**
default

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```json
{
  "rules": {
    "comma-spacing": 2,
    "space-in-parens": [2, "never"]
  }
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

Source code:
```js
fn(a, b, )

// or

fn(a, b,)
```

Here is the reproduction link: https://eslint.org/demo/#eyJ0ZXh0IjoiLyogZXNsaW50IGNvbW1hLXNwYWNpbmc6IDIgKi9cclxuLyogZXNsaW50IHNwYWNlLWluLXBhcmVuczogWzIsICduZXZlciddICovXHJcbmZuKGEsIGIsKSIsIm9wdGlvbnMiOnsicGFyc2VyT3B0aW9ucyI6eyJlY21hVmVyc2lvbiI6OSwic291cmNlVHlwZSI6InNjcmlwdCIsImVjbWFGZWF0dXJlcyI6e319LCJydWxlcyI6e30sImVudiI6e319fQ==

**What did you expect to happen?**
Lint-free.

**What actually happened? Please include the actual, raw output from ESLint.**
Reported linting error, see the reproduction link.

**Are you willing to submit a pull request to fix this bug?**
Yes.

**Comment by ljharb** [comment]
It seems like maybe space-in-parens should ignore trailing function commas (which i presume means you also had ecmaVersion 2018 in your config)
